### PR TITLE
Documenter 1.0 upgrade

### PR DESCRIPTION
**Pull request by ArnoStrouwen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ the documentation, which contains the unreleased features.
   - Integrated logging suite for handling connections to TensorBoard
   - Handling of (partial) integro-differential equations and various stochastic equations
   - Specialized forms for solving `ODEProblem`s with neural networks
-  - Compatability with [Flux.jl](https://docs.sciml.ai/Flux.jl/stable/) and [Lux.jl](https://docs.sciml.ai/Lux/stable/)
+  - Compatability with [Flux.jl](https://docs.sciml.ai/Flux/stable/) and [Lux.jl](https://docs.sciml.ai/Lux/stable/)
```
Comment (ChrisRackauckas) [inline README.md:40]:
```suggestion
  - Compatability with [Flux.jl](https://fluxml.ai/) and [Lux.jl](https://lux.csail.mit.edu/)
```

**Comment by ChrisRackauckas** [comment]
The docs successfully built on master, so I put this on top of a successful master.

**Comment by ChrisRackauckas** [comment]
Hmm I thought I already fixed those links?

**Comment by ArnoStrouwen** [comment]
You missed some and I squashed everything.

**Pull request merged**